kibble/api/films.go

```diff
@@ -252,27 +272,19 @@ type filmV2 struct {
 	Studio []struct {
 		Name string `json:"name"`
 	} `json:"studio"`
-	Overview    string   `json:"overview"`
-	Tagline     string   `json:"tagline"`
-	ReleaseDate string   `json:"release_date,omitempty"`
-	Runtime     float64  `json:"runtime"`
-	Countries   []string `json:"countries"`
-	Languages   []string `json:"languages"`
-	Genres      []string `json:"genres"`
-	Tags        []string `json:"tags"`
-	Title       string   `json:"title"`
-	Slug        string   `json:"slug"`
-	FilmID      int      `json:"film_id"`
-	ID          int      `json:"id"`
-	ImageUrls   struct {
-		Portrait       string `json:"portrait"`
-		Landscape      string `json:"landscape"`
-		Header         string `json:"header"`
-		Carousel       string `json:"carousel"`
-		Bg             string `json:"bg"`
-		Classification string `json:"classification"`
-		Seo            string `json:"seo"`
-	} `json:"image_urls"`
+	Overview        string                      `json:"overview"`
+	Tagline         string                      `json:"tagline"`
+	ReleaseDate     string                      `json:"release_date,omitempty"`
+	Runtime         float64                     `json:"runtime"`
+	Countries       []string                    `json:"countries"`
+	Languages       []string                    `json:"languages"`
+	Genres          []string                    `json:"genres"`
+	Tags            []string                    `json:"tags"`
+	Title           string                      `json:"title"`
+	Slug            string                      `json:"slug"`
+	FilmID          int                         `json:"film_id"`
+	ID              int                         `json:"id"`
+	ImageUrls       map[string]interface{}      `json:"image_urls"`
```

Comment:
Any reason to use an interface over string type like `map[string]string`?

We can see, it is useful to handle null values (eg: https://go.dev/play/p/QwnEs5T95Tf)
But are we taking advantage of this null anywhere?

In [here](https://github.com/shift72/kibble/blob/AB%235032-film-imagemap/kibble/api/films.go#L250), we check for both null and empty strings. If it is a string type, we only have to check for emptiness.